services/admin/src/server.ts

```diff
@@ -568,6 +568,20 @@ export function buildApp({ workspace }: { workspace: Workspace }): Application {
     }
   });
 
+  app.post<
+    NoParams,
+    Admin.GenerateCardPinResponse,
+    Admin.GenerateCardPinRequest
+  >('/admin/pins/generate-card-pin', (_request, response) => {
```

Comment:
I'd do this like so to be more standard REST and as PINs should be unique per-election:

```suggestion
  >('/admin/:electionId/pins', (_request, response) => {
```

Reply:
ah, i didn't think to ask about whether we need to scope uniqueness to a single election! how does that tie in with system admin cards being (IIRC) election-agnostic?

Reply:
cc @arsalansufi, in case this came up in earlier discussions about unique PINs

Reply:
Ah yes great q! The original intent was to have the uniqueness be scoped to a single election. (Or rather, that's all we need for VVSG2. There's no requirement that PINs be unique across elections, though maybe @mattroe can confirm.)

In terms of how this interplays with sys admin cards being election-agnostic, the ability to program election manager and poll worker cards is only made visible once a sys admin has configured the machine for a specific election, so we'll have an election ID available. (Zooming out, the sys admin configures the election definition on VxAdmin and programs election manager cards for that specific election. Election managers then use their cards to configure the other machines.)

Reply:
If we're going to need to call this endpoint outside the context of an election for sysadmin cards, then I can see why we'd want to leave it as you had it. Product-wise as long as we're okay with PINs being unique across all elections on a machine then this works as-is. @mattroe?

Reply:
> If we're going to need to call this endpoint outside the context of an election for sysadmin cards

Ah yes of course - forgot about that case